project_app/components/ToolBar.js

```diff
@@ -1,45 +1,23 @@
-import styled from "styled-components";
 import Image from "next/image";
+import { StyledContainer } from "./styled";
 
 function ToolBar() {
   return (
-    <Container>
-      <div className="search">
-        <div>
-          <input type="text" placeholder="Search board game" />
-          <div
-            style={{
-              position: "absolute",
-              width: "18.99px",
-              height: "18.73px",
-              top: "170px",
-              left: "980px",
-            }}
-          >
-            <Image src="/search.png" fill="absolute" alt="image" />
-          </div>
-        </div>
-      </div>
-    </Container>
+    <div>
+      <StyledContainer>
+        <input type="text" placeholder="Search board game" />
+        <section
+          style={{
+            top: "170px",
+            left: "970px",
+            position: "absolute",
```

Comment:
why do we still have this inline styles?

Reply:
sorry look at this commit @idiglove